docs/architecture.md

```diff
@@ -75,3 +116,114 @@ the decision.
 ### Attestation diagram
 
 ![attestation](images/Fobnail-flows-attestation.png)
+
+---
+
+## Keys and certificates
+
+### Platform Owner
+
+#### Knows in advance
+
+- Platform Owner key (signing and encryption)
+  - Fobnail and remote platform provisioning
+  - used for TLS and for signing certificates
+- Platform Owner certificate
+  - Fobnail and remote platform provisioning
+  - has to be able to provide whole CA chain
+  - both Fobnail Token and Platform must trust root CA in this chain
+- root CA certificate for EK certificate chain
+  - remote platform provisioning
+
+#### Receives
+
+- Fobnail Identity key
+  - Fobnail provisioning
+- Fobnail Encryption key
+  - Fobnail provisioning
+- EK certificate
+  - remote platform provisioning
+- AIK
+  - remote platform provisioning
+
+#### Produces
+
+- Fobnail Identity certificate
+  - Fobnail provisioning
+- Fobnail Encryption certificate
```

Comment:
Do we need encryption certificate at all?
We could use CSR to avoid sending key itself.
We could do the same for identity certificate.